### PR TITLE
Add attributes and attribute test

### PR DESCRIPTION
**Pull request by chrisstankus99**
Add attributes and attribute test
Add truncated files

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  def self.from_csv(locations)
+    game_path = locations[:games]
+    team_path = locations[:teams]
+    game_teams_path = locations[:game_teams]
+
+    StatTracker.new(game_path, team_path, game_teams_path)
+  end
+
+  attr_reader :game_path, :team_path, :game_teams_path
```
Comment (kelshadarby) [inline lib/stat_tracker.rb:11]: Might want to put a space in between the attr_reader and initialize

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  def initialize(game_path, team_path, game_teams_path)
+    @game_path = game_path
```
Comment (foleymichelle9) [inline lib/stat_tracker.rb:13]: Great names! Makes it easy to read :)

File: test/stat_tracker_test.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+require_relative 'test_helper'
```
Comment (margoflewelling) [inline test/stat_tracker_test.rb:1]: super helpful file!

File: lib/stat_tracker.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+class StatTracker
+
+  def self.from_csv(locations)
+    game_path = locations[:games]
+    team_path = locations[:teams]
+    game_teams_path = locations[:game_teams]
+
+    StatTracker.new(game_path, team_path, game_teams_path)
```
Comment (kelshadarby) [inline lib/stat_tracker.rb:8]: Love the initialization in from_csv

**Pull request merged**